api/envoy/extensions/load_balancing_policies/override_host/v3/override_host.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+package envoy.extensions.load_balancing_policies.override_host.v3;
+
+import "envoy/config/cluster/v3/cluster.proto";
+import "envoy/type/metadata/v3/metadata.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.load_balancing_policies.override_host.v3";
+option java_outer_classname = "OverrideHostProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/load_balancing_policies/override_host/v3;override_hostv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Override Host Load Balancing Policy]
+// [#extension: envoy.load_balancing_policies.override_host]
+
+// Configuration for the Override Host Load Balancing policy.
+//
+// This policy allows endpoint picking to be implemented in an external to Envoy
+// component. While it can work with a built in HTTP filter it is primarily intended to
+// work in conjunction with the ext_proc or WASM HTTP filters.
+//
+// The ext_proc or WASM filter must implement k8s proposal for AI gateway inferences extensions
+// https://github.com/kubernetes-sigs/gateway-api-inference-extension/tree/main/docs/proposals/004-endpoint-picker-protocol
+// to integrate with this load balancing policy.
+//
+// This policy uses selected endpoints from either request headers or request metadata.
+//
+// The endpoint picker protocol for k8s gateway inference extensions uses the
```

Comment:
Similarly, please change the wording here to describe that the value of the specified header must be an IP:port string that matches an existing endpoint, rather than describing it specifically in terms of the k8s inference gateway API.